client/src/actions/actions.js

```diff
@@ -33,19 +36,30 @@ export const CALL_SENT = 'CALL_SENT';
 import { push } from 'react-router-redux';
 
 export function fetchEntries() {
+  let config = {
+    method: 'GET',
+    headers: {
+      authorization: 'Bearer ' + localStorage.getItem('id_token')
```

Comment:
Kind of late for me to be mentioning this i guess but maybe we should call the token reflective_token or something to namespace our app. id_token is a little generic, might have clashes with other apps.